public/app/features/trails/ActionTabs/BreakdownScene.tsx

```diff
@@ -74,12 +79,64 @@ export class BreakdownScene extends SceneObjectBase<BreakdownSceneState> {
       }
     });
 
+    const trail = getTrailFor(this);
+    trail.state.$timeRange?.subscribeToState(() => {
+      this.clearBreakdownPanelAxisValues();
```

Comment:
```suggestion
      // The change in time range will cause a refresh of panel values,
      // so we clear the axis range so it can be recalculated when the calls
      // to `reportBreakdownPanelData` start being made from the panels' behavior.       
      this.clearBreakdownPanelAxisValues();
```